README.md

```diff
@@ -449,35 +465,48 @@ To view the message in a queue, login to [rabbitmq management console](http://lo
 ![binding](docs/document.ready.bind.png)
 
 ####if you want to run the sample app using sftp do the following:
-Create a sftp server container
+
+step 1. Create a sftp server container (from WindowsPowerShell or GitBash)
 ```bash
 docker run -p 22:22 -d atmoz/sftp myname:pass:::upload
 ```
-User "myname" with password "pass" can login with sftp and upload files to a folder called "upload". We are forwarding the container's port 22 to the host's port 22.
-Use a Sftp Client application ( such as Fillzilla, WinSCP, coreFTP) to connect to the server.(use sftp protocal and ip: localhost, port:22)
-update the [application.yml](jrcc-access-spring-boot-sample-app/src/main/resources/application.yml)
+
+step 2. User "myname" with password "pass" can login with sftp and upload files to a folder called "upload". We are forwarding the container's port 22 to the host's port 22.
+
+step 3. Use a Sftp Client application ( such as Fillzilla, WinSCP, coreFTP) to connect to the server.(use sftp protocal and ip: localhost, port:22)
```

Comment:
supernit: should it be `filezilla`? also `protocol` 
```suggestion
step 3. Use a Sftp Client application ( such as Filezilla, WinSCP, coreFTP) to connect to the server (use sftp protocol and ip: localhost, port:22).
```

Reply:
nice catch.